pilot/pkg/bootstrap/server.go

```diff
@@ -335,10 +336,19 @@ func (s *Server) initKubeClient(args *PilotArgs) error {
 			needToCreateClient = true
 		case EurekaRegistry:
 			needToCreateClient = true
+		case CloudFoundryRegistry:
+			//valid registry, but does not need to create client
+			continue
+		default:
+			errs = multierror.Append(errs, fmt.Errorf("Service registry %s is not supported.", r))
```

Comment:
Remove the `errs` and `default` case. Right now if you had k8s + Consul, it would fail because it won't accept `ConsulRegistry`.